v1/content.go

```diff
@@ -0,0 +1,39 @@
+package statusproto
+
+import (
+	"github.com/gomarkdown/markdown"
+	"github.com/gomarkdown/markdown/ast"
+	"strings"
+	"unicode"
+	"unicode/utf8"
+)
+
+// Content contains the chat ID and the actual text of a message.
+type Content struct {
+	ChatID     string   `json:"chat_id"`
+	Text       string   `json:"text"`
+	ResponseTo string   `json:"response-to"`
+	Name       string   `json:"name"` // the ENS name of the sender
+	ParsedText ast.Node `json:"parsedText"`
+	LineCount  uint     `json:"lineCount"`
```

Comment:
Any reason to use `uint` instead of `int`?

Reply:
just because it can't be negative, should I change it to int instead?

Reply:
Can it be realistically set to a negative number? `len(slice)` also can't be negative but it uses `int`. If you are fine with type casting and `unint` type carries some useful information that is missing when using `int`, let's keep it.

Reply:
no, will never be negative, i just went instinctively for uint, what s more customary in this case? happy to go for the one feels more natural ( seens like int is)

Reply:
More natural would be `int` because its name is `LineCount` so we know it's only positive. If it was a continuous counter like a metric counter, `int64` would be better. If a counter was serialized to bytes and send through the network, then I would go with `int32` or `int64` or even unsigned ints.

But I actually don't know the reason why `len()` returns `int`. The only thing that comes to my mind is convenience. Dealing with unsigned ints is harder and can result in odd values in the case of overflow.